internal/storage/storage.go

```diff
@@ -160,8 +160,8 @@ func (s *Storage) GetHostses() ([]Hosts, error) {
 }
 
 // cluster
-func (s *Storage) InsertCluster(name, uuid, description, topology string) error {
-	return s.write(InsertCluster, uuid, name, description, topology)
+func (s *Storage) InsertCluster(name, uuid, description, topology string, deployType string) error {
```

Comment:
`name, uuid, description, topology, deployType string` is better

Reply:
I've fixed this.